### PR TITLE
Exploratory analysis of NASDAQ data

### PR DESCRIPTION
**Pull request by edoardomocchi**
uploaded df on pickle file.
anlysis and plot on jupnotebook file called task1.
checkdata nel folder log.

### CODE REVIEW

**Review by giutala — CHANGES_REQUESTED** [review]
- do not name the file task1, name it after what it does 
- use helpermodules to pickle and clean the data: delete checkdata and make a main where we use the helpermodules 
- add conclusions to each thing you do on the notebook (check riccardo polo's notebook on clustering project) 
- why is the cointegration p value 1? make sure it is correct

File: main.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 from helpermodules.df_dataretrieval import IndexData_Retrieval
 from helpermodules.correlation_study import CorrelationAnalysis
 
-df_nasdaq = IndexData_Retrieval(filename='nasdaq_dataframe', link='https://en.wikipedia.org/wiki/Nasdaq-100',months=2, frequency='1min')
+df_nasdaq = IndexData_Retrieval(filename='nasdaq_dataframe', link='https://en.wikipedia.org/wiki/Nasdaq-100',months=2, frequency='1min') # Edoardo months and frequency?
```
Comment (giutala) [inline main.py]: frequency daily, months three years

**Review by giutala — CHANGES_REQUESTED** [review]
frequency and months, update

**Review by giutala — CHANGES_REQUESTED** [review]
- do not push .gitmodules 
- make sure helpermodules is the latest version by running git submodule update --remote on the terminal

**Review by ErTucano — APPROVED** [review]

**Pull request closed without merging**